data/json/items/armor/gloves.json

```diff
@@ -948,5 +948,27 @@
     "environmental_protection": 2,
     "material_thickness": 5,
     "flags": [ "STURDY", "OUTER", "RAINPROOF", "ONLY_ONE" ]
+  },
+  {
+    "id": "gloves_studded",
+    "type": "ARMOR",
+    "name": { "str": "pair of studded gloves", "str_pl": "pairs of studded gloves" },
+    "description": "A pair of gloves with studded metal knuckles.",
+    "weight": "218 g",
+    "volume": "250 ml",
+    "price": 1100,
+    "price_postapoc": 250,
+    "bashing": 2,
+    "cutting": 1,
+    "material": [ "leather", "cotton" ],
+    "symbol": "[",
+    "looks_like": "gloves_leather",
+    "color": "light_gray",
+    "covers": [ "HANDS" ],
+    "coverage": 100,
+    "encumbrance": 2,
```

Comment:
I'd increase encumbrance a bit to distinguish them from ordinary leather gloves.